modules/Microsoft.Web/connections/readme.md

```diff
@@ -180,7 +180,7 @@ module connections './Microsoft.Web/connections/deploy.bicep' = {
     name: 'azuremonitor'
     // Non-required parameters
     connectionApi: {
-      id: '/subscriptions/<<subscriptionId>>/providers/Microsoft.Web/locations/westeurope/managedApis/azuremonitorlogs'
+      id: '/subscriptions/${subscription().subscriptionId}/providers/Microsoft.Web/locations/westeurope/managedApis/azuremonitorlogs'
```

Comment:
Must be updated once template is

Reply:
The connection readme file still needs to be updated